### PR TITLE
Add ColorPicker to docs

### LINKED ISSUE #1743: ColorPicker component does not appear in documentation
### Describe the bug

The color picker component does not show up in the documentation. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Go to gradio.app/docs/

### Screenshot

image

### Logs

```shell
-
```

### System Info

```shell
-
```

### Severity

annoying

### PR DESCRIPTION
**Pull request by freddyaboulton**
# Description

Fixes #1743

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: demo/color_picker/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Pillow
```
Comment (freddyaboulton) [inline demo/color_picker/requirements.txt:1]: Confirmed the color picker demos is properly built on the website demos

Screen Shot 2022-07-12 at 12 40 32 PM

File: gradio/components.py [hunk 2]
```diff
@@ -2905,12 +2906,12 @@ def __init__(
     ):
         """
         Parameters:
-        value (str): default text to provide in color picker.
-        label (Optional[str]): component name in interface.
-        show_label (bool): if True, will display label.
-        interactive (Optional[bool]): if True, will be rendered as an editable color picker; if False, editing will be disabled. If not provided, this is inferred based on whether the component is used as an input or output.
-        visible (bool): If False, component will be hidden.
-        elem_id (Optional[str]): An optional string that is assigned as the id of this component in the HTML DOM. Can be used for targeting CSS styles.
+            value: default text to provide in color picker.
```
Comment (freddyaboulton) [inline gradio/components.py:2909]: Some minor tweaks here to conform to the docstring format

File: demo/color_picker/requirements.txt (hunk 1 above)
Comment (abidlabs) [reply demo/color_picker/requirements.txt:1]: Why do we need this requirements file, when `pillow` is a dependency of `gradio`?

Comment (abidlabs) [reply demo/color_picker/requirements.txt:1]: Why do we need this requirements file, when `pillow` is a dependency of `gradio`?

**Comment by abidlabs** [comment]
LGTM, just wondering about the requiements.txt file

File: demo/color_picker/requirements.txt (hunk 1 above)
Comment (aliabid94) [reply demo/color_picker/requirements.txt:1]: in case gradio changes its dependencies.

**Comment by aliabid94** [comment]
LGTM

**Review by aliabid94 — APPROVED** [review]

**Pull request merged**